alethzero/MainWin.cpp

```diff
@@ -860,8 +860,8 @@ void Main::readSettings(bool _skipGeometry)
 		p->readSettings(s);
 	});
 
-	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setAsk(u256(s.value("askPrice", "500000000000").toString().toStdString()));
-	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setBid(u256(s.value("bidPrice", "500000000000").toString().toStdString()));
+	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setAsk(u256(s.value("askPrice", "50000000000").toString().toStdString()));
+	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setBid(u256(s.value("bidPrice", "50000000000").toString().toStdString()));
```

Comment:
I want to use `c_defaultGasPrice` here. But he doesn't accept a u256, or a string and I cant convert a string into a QString. What is the common way of providing a u256 here?